iothub_client/samples/iothub_client_sample_upload_to_blob_mb/iothub_client_sample_upload_to_blob_mb_stress.c

```diff
@@ -63,20 +68,20 @@ static IOTHUB_CLIENT_FILE_UPLOAD_GET_DATA_RESULT getDataCallback(IOTHUB_CLIENT_F
                 *data = NULL;
                 *size = 0;
 
-                (void)printf("Indicating upload is complete (%d blocks uploaded)\r\n", blocks_sent);
+                (void)printf("\nIndicating upload is complete (%d blocks uploaded)\r\n", blocks_sent);
             }
         }
         else
         {
             // The last call to this callback is to indicate the result of uploading the previous data block provided.
             // Note: In this last call, data and size pointers are NULL.
 
-            (void)printf("Last call to getDataCallback (result for %dth block uploaded: %s)\r\n", blocks_sent, MU_ENUM_TO_STRING(IOTHUB_CLIENT_FILE_UPLOAD_RESULT, result));
+            (void)printf("\nLast call to getDataCallback (result for %dth block uploaded: %s)\r\n", blocks_sent, MU_ENUM_TO_STRING(IOTHUB_CLIENT_FILE_UPLOAD_RESULT, result));
```

Comment:
What is the issue with all the \n? This is diverging from the usual style on the samples.

Reply:
This is a stress test / sample that will only be available in this branch for a particular customer.
The \n is to add a new line after the progress done using "." characters.